### ISSUE TITLE
Segmented reports below the actual report

### ISSUE THREAD

**Issue by tsteur** [piwik/piwik#9692]
We could show segmented report views of the actual reports. For example first we show the "Page URL" report.

Below we show the top 5 urls with different segments. Each segment in it's own box. This allows you eg to quickly see how the top 5 url is different for each country, each browser, each operating system, ... Below the 5 urls would be "more details" link and it would either show more urls or change the main report to show all urls applied by this segment. 

We could have some predefined segments per report and/or let users customize which segments to apply to get a better overview.

To not pollute the UI it could be hidden or disabled by default and when clicking a button or changing visualization it would be displayed.

It's a bit like the "Comparison Dashboard" plugin but for reports.

**Comment by gaumondp** [comment]
I'm not totally sure I understand this proposition. Do you mean I go to Actions=>Pages and at the bottom I can see 5 different boxes with: 

URL 1
Top 5 countries
Top 5 browsers
Top 5 Devices

Then URL2 and so on ?

**Comment by tsteur** [comment]
Nope, you'd see top 5 urls by country, top 5 urls by browsers, top 5 urls by devices ...

We could show them a bit smaller and like 2 boxes per row. Depending on the applied segments and on the use case of your website or app this can be very useful.

**Comment by gaumondp** [comment]
It's more obvious now.

Those "Top 5" (or Top 10 whatever) remind me a bit my request for an annual (or better, selected period ) report made in #9435 . Not a duplicate, really just the same need for "Top Something" kinda reports.
